lua/oil/config.lua

```diff
@@ -66,7 +66,7 @@ local default_config = {
     ["<C-h>"] = { "actions.select", opts = { horizontal = true }, desc = "Open the entry in a horizontal split" },
     ["<C-t>"] = { "actions.select", opts = { tab = true }, desc = "Open the entry in new tab" },
     ["<C-p>"] = "actions.preview",
-    ["<C-c>"] = "actions.close",
+    ["<C-c>"] = { "actions.close", opts = { exit_if_last_buf = false }, desc = "Close oil, optionally exit vim if oil is closed as last buffer" },
```

Comment:
Let's not change the default close action. The option will be automatically documented in `:help actions.close`. If we tried to put every parameter in the README just for documentation purposes it would be kind of a mess.